### PR TITLE
Gridded the input-output section.

### PR DESCRIPTION
**Pull request by varnan6**
Major highlights:
    - Griddded the input-output section,
    - Added a display-on-hover effect on the parameter-description of each parameter when hovered on the ⓘ text,
    - Aligned the title to the left,
    - Designed hover effect on Github anchor tags, and
    - Concised the parameter-description.

Future agenda:
    - Add masking effect to title,
    - Create an input form design similar to the "Sentimentinator" input form but with different theme,
    - Center the output section,
    - **On priority: Arrange the parameter-description sections to their respective ⓘ  text with display: absolute**, and
    - Add responsiveness to the site.

### CODE REVIEW

File: static/css/index.css [hunk 1]
```diff
@@ -24,12 +24,20 @@ body {
 }
 
 .title {
-  font-size: 3rem;
+  font-size: 7.2rem;
```
Comment (Hardvan) [inline static/css/index.css:27]: Humongous title size, noice

File: templates/index.html [hunk 2]
```diff
@@ -29,157 +29,179 @@ <h1 class="title">PolyPersona</h1>
...
-        models from the Gemini API. Users input a celebrity's name and desired
-        message, and the system generates responses resembling the chosen
-        personality's speech patterns. The application offers multi-language
-        support, customizable model parameters, and audio generation from the
-        translated text. Implemented with Flask, HTML/CSS/JavaScript, and
-        Python, PolyPersona provides a user-friendly interface for easy
-        interaction. Install the application by cloning the repository, setting
-        up dependencies, and running the server locally. Try it out to
-        experience personalized celebrity-style responses!
+        models from the Gemini API. <br /><br />Users input a celebrity's name
```
Comment (Hardvan) [inline templates/index.html:32]: Nice use of line breaker tags

**Review by Hardvan — APPROVED** [review]
Looks good

File: static/css/index.css (hunk 1 above)
Comment (varnan6) [reply static/css/index.css:27]: 😂

**Pull request merged**